### PR TITLE
[IOPID-694] : noProfile component in profile page

### PR DESCRIPTION
**Pull request by OraldoDoci**
## Short description

Moved /nessun-profilo Page into Profile Page as a component 

## List of changes proposed in this pull request

- Moved /nessun-profilo Page into Profile Page as a component

## How to test

1. Login as L2
2. Mock the GetProfile Api response with Moockon to return status code 404

### CODE REVIEW

File: src/app/[locale]/page.tsx [hunk 1]
```diff
@@ -24,110 +26,124 @@ const Profile = () => {
...
-                    : 'L’accesso a IO è bloccato con livello di sicurezza 2'
-                }
-                placement="top"
-                arrow
-              >
-                <HelpOutlineIcon color="primary" />
-              </Tooltip>
+  return (
+    <>
+      {profileIsAvailable === false ? (
```
Comment (shadowsheep1) [inline src/app/[locale]/page.tsx:54]: Do we usually do the explicit check for boolean? I mean, why not use `!profileIsAvailable`

File: src/app/[locale]/page.tsx [hunk 2]
```diff
@@ -10,12 +10,14 @@ import { ProfileCards } from './_component/profileCards/profileCards';
...
 import { commonBackground } from './_utils/styles';
 import { storageUserOps } from './_utils/storage';
+import { NoProfile } from './_component/noProfile/noProfile';
 import { WebProfileApi } from '@/api/webProfileApiClient';
 import { SessionState } from '@/api/generated/webProfile/SessionState';
 
 const Profile = () => {
   const [profileData, setProfileData] = useState<ProfileData>();
   const [sessionData, setSessionData] = useState<SessionState>();
+  const [profileIsAvailable, setProfileIsAvailable] = useState<boolean>(false);
```
Comment (shadowsheep1) [inline src/app/[locale]/page.tsx:20]:
```suggestion
  const [isProfileAvailable, setIsProfileAvailable] = useState<boolean>(false);
```

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: src/app/[locale]/page.tsx (hunk 1 above)
Comment (OraldoDoci) [reply src/app/[locale]/page.tsx]: Thought it was easier to read. Changed the code !isProfileAvailable.

**Review by shadowsheep1 — APPROVED** [review]
LGTM 

https://github.com/pagopa/io-web-profile/assets/16268789/8656c39e-1506-4005-94db-9beb6edc706b

**Pull request merged**